libs/ui/accordion/brain/src/lib/brn-accordion-trigger.component.ts

```diff
@@ -38,15 +63,22 @@ export class BrnAccordionTriggerDirective {
 		this._hostPressedListener.subscribe(() => {
 			this._accordion.toggleItem(this._item.id);
 		});
+	}
 
-		fromEvent(this._elementRef.nativeElement, 'focus')
-			.pipe(takeUntilDestroyed())
-			.subscribe(() => {
-				this._accordion.setActiveItem(this);
-			});
+	ngOnInit(): void {
+		fromEvent(this.trigger.nativeElement, 'focus')
```

Comment:
question: how do you handle the unsubscribe from the `fromEvent` Observable?

Reply:
The takeUntilDestroyed operator takes care of that. More here: https://angular.io/api/core/rxjs-interop/takeUntilDestroyed

Reply:
Crap, I just missed that line. :)